unite_contranges.py

```diff
@@ -0,0 +1,147 @@
+#Allison Towner
+#09 October 2020
+
+#continuum_selections.py in analysis code is located at:
+#in ALMA_IMF/reduction/analysis/
+
+import numpy as np
+execfile('/orange/adamginsburg/atowner/ALMA_IMF/contranges/parse_contdotdat.py')
+
+#Read in metadata from .json file
+basepath = '/orange/adamginsburg/ALMA_IMF/2017.1.01355.L'
+with open(basepath + '/contdatfiles.json', 'r') as fh:
+    contdatfiles = json.load(fh)
+with open(basepath + '/metadata.json', 'r') as fh:
+    metadata = json.load(fh)
+
+
+#Set field and band name - this should be detected automatically in the final version of the script but is hardwired here for testing purposes
+field = 'G327.29'
+band = 'B3'
+
+#use parse_contdotdat to read the 12m-long, 12m-short, and 7m contranges into arrays from their cont.dat files
+#NOTE: This part of the script assumes there are only three cont.dat files, and they are in a specific order!
+#This should be the same across all metadata but if it is not this part of the script will need to be altered.
```

Comment:
This may not hold for several fields; there are some fields with only one 12m configuration

Reply:
Ok. I think I have a solution to that but give me a few minutes to implement - I will need to track and and change the list names throughout the code.